engines/scumm/scumm.cpp

```diff
@@ -246,7 +246,7 @@ ScummEngine::ScummEngine(OSystem *syst, const DetectorResult &dr)
 		_debugMode = true;
 
 	_copyProtection = ConfMan.getBool("copy_protection");
-	if (ConfMan.getBool("demo_mode"))
+	if (ConfMan.getBool("demo_mode")||ConfMan.getBool("enable_demo_mode"))
```

Comment:
Could you please write it like this?
`if (ConfMan.getBool("demo_mode") || ConfMan.getBool("enable_demo_mode"))` 
(with spaces before and after the `||` operator)

Reply:
Done, also squashed into a single commit.